app/models/domain_block.rb

```diff
@@ -12,11 +12,12 @@
 #
 
 class DomainBlock < ApplicationRecord
-  enum severity: [:silence, :suspend]
+  enum severity: [:silence, :suspend, :media_only]
```

Comment:
So we're ok with admins creating domain blocks that are complete no-ops? I guess it doesn't hurt anything, but it seems a little silly...